### PR TITLE
Differentiate between functions and classes

### PR DESCRIPTION
**Pull request by RJ722**
Closes: #61 

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation in the README file.
- [ ] I have updated the documentation accordingly.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: vulture/core.py [hunk 1]
```diff
@@ -206,7 +207,8 @@ def file_lineno(item):
         unused_item_found = False
         for item in sorted(
                 self.unused_funcs + self.unused_imports + self.unused_props +
-                self.unused_vars + self.unused_attrs, key=file_lineno):
+                self.unused_classes + self.unused_vars + self.unused_attrs,
+                key=file_lineno):
```
Comment (RJ722) [inline vulture/core.py]: Should we introduce a new variable `all = self.unused_funcs + self.unused_imports + self.unused_props + self.unused_classes + self.unused_vars + self.unused_attrs` ?

File: tests/test_scavenging.py [hunk 2]
```diff
@@ -159,8 +163,9 @@ class Bar():
 b = Bar()
 """)
     assert v.used_attrs == []
-    assert v.defined_funcs == ['Bar']
-    assert v.unused_funcs == []
+    assert v.defined_classes == ['Bar']
+    assert v.unused_classes == []
+    assert v.unused_vars == ['b']
```
Comment (jendrikseipp) [inline tests/test_scavenging.py]: Nice catch!

File: vulture/core.py (hunk 1 above)
Comment (jendrikseipp) [inline vulture/core.py]: But maybe this is useful for external tools like VultureBear. I'll add a method.

**Comment by jendrikseipp** [comment]
Nice work!

**Comment by RJ722** [comment]
Thank You! 😄

**Pull request merged**